tests/client_test.py

```diff
@@ -111,6 +142,35 @@ def test_get_client_kwargs_login_customer_id_as_None(self):
                     "endpoint": None,
                     "login_customer_id": None,
                     "logging_config": None,
+                    "linked_customer_id": None,
+                },
+            )
+
+    def test_get_client_kwargs_linked_customer_id_as_None(self):
```

Comment:
Nit: I don't think you need to spell None with a capital letter.

Reply:
Fix in both places where this occurs.